### DISCUSSION TITLE
About the three modes of IP Masquerade Mode

### DISCUSSION THREAD

**Discussion by huzesama** [Mygod/VPNHotspot#570] [Q&A]
when I try to switch between these modes, my app often cannot connect to the server, and sometimes it suddenly works.

**Comment by Mygod** [comment]
Export debug information?

**Reply by huzesama** [reply]
emmm....Sorry, I seem to have misunderstood something. I mistook the switch for sharing VPN via USB (or WiFi) as the monitoring switch.
But there are still some things I don't understand. About the functions (operating mechanisms) of the three modes of IP Masquerade Mode. Because when I try to switch between these modes, my app often cannot connect to the server, and sometimes it suddenly works.
I have updated my discussion.
